processor/src/main/java/org/mapstruct/ap/internal/processor/MapperCreationProcessor.java

```diff
@@ -5,14 +5,24 @@
  */
 package org.mapstruct.ap.internal.processor;
 
+import static javax.lang.model.element.Modifier.FINAL;
```

Comment:
don't try to re-org the imports (nitty-witty thing.. but helps to review the code 😄

Reply:
Do you want me to rebase and keep the order of imports ?